py-polars/polars/plugins.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from polars._utils.unstable import unstable
+
+__all__ = ["get_shared_lib_location"]
+
+
+@unstable()
+def get_shared_lib_location(package_init_path: str) -> str:
+    """
+    Get location of Shared Object file.
```

Comment:
It's not just Shared Object (`.so`) files, right? We're also looking for `.pyd` or `.dll`.

Reply:
Maybe call it dynamic libarry? I think we can see that as overarching the details implementations of windows/linux etc.